src/applications/utility/login/index.tsx

```
@@ -1,61 +1,79 @@
+          alt=""
+        />
+      </_.tempImage>
+      <_.tempBulk />
+      <_.tempMainStyle>
+        <_.tempInputs>
+          {inputList.map((item) => (
+            <Inputs
+              {...item}
+              flex={true}
+            />
+          ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)